test/metabase/driver/sql_jdbc_test.clj

```diff
@@ -24,62 +24,6 @@
                          {:name table, :schema "PUBLIC", :description nil}))}
          (driver/describe-database :h2 (mt/db)))))
 
-(deftest ^:parallel describe-table-test
-  (is (= {:name   "VENUES"
-          :schema "PUBLIC"
-          :fields #{{:name              "ID"
-                     :database-type     "BIGINT"
-                     :base-type         :type/BigInteger
-                     :pk?               true
-                     :database-position 0
-                     :database-required false
-                     :database-is-auto-increment true
-                     :json-unfolding    false}
-                    {:name              "NAME"
-                     :database-type     "CHARACTER VARYING"
-                     :base-type         :type/Text
-                     :database-position 1
-                     :database-required false
-                     :database-is-auto-increment false
-                     :json-unfolding    false}
-                    {:name              "CATEGORY_ID"
-                     :database-type     "INTEGER"
-                     :base-type         :type/Integer
-                     :database-position 2
-                     :database-required false
-                     :database-is-auto-increment false
-                     :json-unfolding    false}
-                    {:name              "LATITUDE"
-                     :database-type     "DOUBLE PRECISION"
-                     :base-type         :type/Float
-                     :database-position 3
-                     :database-required false
-                     :database-is-auto-increment false
-                     :json-unfolding    false}
-                    {:name              "LONGITUDE"
-                     :database-type     "DOUBLE PRECISION"
-                     :base-type         :type/Float
-                     :database-position 4
-                     :database-required false
-                     :database-is-auto-increment false
-                     :json-unfolding    false}
-                    {:name              "PRICE"
-                     :database-type     "INTEGER"
-                     :base-type         :type/Integer
-                     :database-position 5
-                     :database-required false
-                     :database-is-auto-increment false
-                     :json-unfolding    false}}}
-         (driver/describe-table :h2 (mt/db) (t2/select-one Table :id (mt/id :venues))))))
-
-(deftest ^:parallel describe-table-fks-test
```

Comment:
There's nothing this covers that `describe-table-fks-test` in `test/metabase/driver_test.clj` doesn't cover [here](https://github.com/metabase/metabase/pull/41528/files#diff-a1b51f96feece082d660d8d97b049f2ebad66dfb2de5a095ac8ff848fac59e38R234).